prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Francis Manalang';
+    console.log('Full Name:', fullName);
+    var numberOfPets = '1';
+    console.log('Number of Pets:', numberOfPets);
+    var likesVegetables = 'False';
+    console.log('Likes Vegetables:', likesVegetables);
+    var nothing = 'null';
+    console.log('nothing:', nothing);
```

Comment:
Hi Francis, be sure to leave out the quotes from numbers, booleans, and null so that the browser doesn't interpret them as strings

Reply:
Hi Tim, 
Got it. Please let me know if you got my changes. Thanks!